spec/rubocop/cop/style/trivial_accessors_spec.rb

```diff
@@ -349,6 +349,24 @@
       end
     end
 
+    context 'explicit receiver reader with instance_eval' do
+      let(:source) do
+        [
+          'obj.instance_eval do |_|',
+          '  def foo',
+          '    @foo',
+          '  end',
+          'end'
+        ]
+      end
+
+      it 'does not autocorrect' do
```

Comment:
You shouldn't care about auto-correct at all - simply check there are no offenses.